### ISSUE TITLE
Remove non-standard CWs

### ISSUE THREAD

**Issue by SoniEx2** [mastodon/mastodon#20342]
### Pitch

Mastodon should remove the CW box and the "show more" button on CWs. For backwards compatibility it should always show the CW text. For cases where it's important, namely porn, mastodon should make available a standardized, translatable set of CWs.

It should still be possible to hide/blur/etc posts by keywords, ofc.

### Motivation

Having a CW box puts the onus(?) of setting up CWs on the poster.

Not having it puts the onus of filtering posts on the reader.

So-called "less-than-ideal" solutions may be necessary for harm reduction, sometimes. We strongly believe this is the better approach.

TL;DR: this "fixes" racism.

**Comment by DRKSLV** [comment]
I think the flexibility of Content Warnings makes them incredibly useful.
There are many reasons one would choose to hide posts and media behind a click...
Maybe it contains discussion of sensitive topics, e.g self harm or eating disorders
Maybe the post has a non-standard charachter limit and is just really large.

I don't think it's worth attempting to standardize CWs by taking choices away from posters.
The benefit of localizing these is pretty small i think.
If it is a Japanese CW, the Posts text is likely also in Japanese, making it as incomprehensible as the cw

And for images, it's simply advisable to not click through CW's you don't understand/haven't bothered to paste into a translation tool

TL;DR: You're asking for the software to be made less flexible, to just make it worse lmao

**Comment by DRKSLV** [comment]
Integrated translation is a feature that's being worked on.... i think...

And would also solve this issue

**Comment by SoniEx2** [comment]
at least ppl would stop demanding other ppl to put things in the CW box when they can just use a filter.

**Comment by Rexogamer** [comment]
I don't get the problem you're tying to solve - if someone asks you to CW something, just do so in future? Filtering can only do so much (for example, slightly different spellings of a word) - in addition, adding CWs is a fairly common request/courtesy on Mastodon, and while it's a shift from platforms like Twitter you usually shouldn't take requests for CWs rudely.

I'm also slightly confused in regards to your last point. Are people actively being racist to you? If so, that's not okay and you should get in touch with your instance's admin/mod team for help. Unfortunately, as Mastodon is decentralised/this is a development repository, we can't do much to help.

**Comment by trwnh** [comment]
you *could* theoretically have something like this:

- separate `summary` from `sensitive`
- allow users to decide whether posts with a summary are collapsed always, collapsed never, or collapsed when sensitive

possible extensions:

- change `sensitive` from a binary boolean flag (true/false) to take into account possible reasons
  - perhaps common reasons can be predefined and prepopulated?
  - i imagine the ui might be something like "check all boxes that apply" or otherwise allowing you to search for tags (not hashtags) to be added

i don't think this issue is the right way to express things, however. the "cw" functionality shouldn't be removed entirely, nor should we be in the business of determining which classifications are "important" (and "namely porn" does not really seem like the primary reason either), and also this doesn't fix racism at all

**Comment by SoniEx2** [comment]
it would mitigate it because you can't complain about ppl not using a feature that doesn't actually exist.

so that's a whole class of racism just gone. simply by removing a feature.

**Comment by DRKSLV** [comment]
🤡

**Comment by SoniEx2** [comment]
ppl who make social networks, be they developers or admins or whatever, are social engineering. if they don't recognize as such they fail to acknowledge their responsibilities, and thus cause immense amounts of harm.

part of those responsibilities is figuring out when stuff isn't actually working out.

**Comment by AlanBell** [comment]
CW can be for current events like a particular election somewhere in the world, or Eurovision 2023 or that SuperbOwl thing, so it would be hard to enumerate a list of official texts in advance.

**Comment by SoniEx2** [comment]
indeed, that's where keywords (or tags) come in handy! they can be filtered out as needed.

**Comment by Nachtpfoetchen** [comment]
With all the different needs and languages,  it is impossible to make a complete list of cw.
If you don't want to use cw don't. But some people need them. Not everyone has the same needs as you.

**Comment by DRKSLV** [comment]
Which is incredibly clunky and cumbersome and readers have to enumerate a long list of tags in advance they'd possibly find disturbing
Maybe you wouldn't click through a CW such as "Violent Dismemberment"
But you'd probably not come up with the idea to specifically filter for #violentDismemberment 

> ppl who make social networks, be they developers or admins or whatever, are social engineering. if they don't recognize as such they fail to acknowledge their responsibilities, and thus cause immense amounts of harm
Thats why they should take care to not remove flexible content warnings in favour of less options

How does having CWs cause harm??
You're living every days like it's opposite day

Your proposal is bad
We're talking "let's remove DMs from twitter, someone called me a dingus there" levels of bad.
You're on some "let's remove turn signals from cars, people always yell at me for not using them" type bull

Someone close this issue 💀
(@Gargron)

**Comment by Rexogamer** [comment]
This might be related to a thread I saw recently about white people asking BIPOC people to CW racism/saying that BIPOC people don't need to tag it for a few reasons? I understand (and agree with!) that thread, but the solution is to ask/tell people not to make these kinds of requests, not to remove a very useful and widely used feature.

**Comment by SoniEx2** [comment]
systemic issues call for systemic changes. how do you propose doing that in a systemic way?

**Comment by DRKSLV** [comment]
It's not a systematic issue though
It's an interpersonal issue.

Someone tells you to hide your activism behind a CW?
Tell then to f* off, possibly block them etc...

I guarantee, having less flexible CWs would be a greater systematic issue.
And i think servers would be reluctant to use new mastodon versions if this was removed....

**Comment by DRKSLV** [comment]
You can't make an issue to shift the opinions and conventions users have when they use your software

**Comment by DRKSLV** [comment]
Ok look.
I do understand and want to take seriously, the tension that exists between activits trying to raise awareness about injustice (often in america), and an existing userbase which i'm guessing was in large part european, and is advocating for CWs like "uspol"

For example: Though i find it interesting, am extremely empathetic towards everyone experiencing e.g the injustice of the American political system...
Learning SO MUCH about it really upsets me, but leaves me without any real action i can take.
I can't vote there, run for office there, protest there, or feasibly light something on fire there.
Raising my awareness is a well-intentioned goal, unfortunately it isn't very useful for you, and i'd better spend my time learning about the state of things in my own nation, where i feel i have the responsibility and ability to take action.

On the other hand, i totally get that raising awareness for an issue and advocating for policies...
It just can't happen behind a CW!
It totally conflicts with the goal of being an activist!
Making your struggle/fight/speech less visible??? 

On the bottom line, weighing both sides, i personally can't say i'd demand of activists to use CWs.
I actually... it's almost disrespectful. But thats my opinion.

Now you, you're comig at this problem from a very wierd angle, which is why i've been responding with so little understanding.
(and i'm sorry)

Look: The CW Field is an accessability feature that people actively depend on. 
Trigger Warnings, Content Warnings for gore, porn..., Disclaimers... you name it.
And you want to take away this crucial feature because...

You can't stand your ground against a few people telling you to CW your activism?
Guess what: People while always come at activists, demanding that they use gentler, less loud, less disruptive ways of protesting.
You don't fix this by "removing the option to be gentle from reality, so people stop asking"
You fix it by standing your ground!! The only job you have as an activist!?! 
This is why i find your proposal so absurd. 

I think i'll not be responding any further. I've said enough already for what this is.

**Comment by CEbbinghaus** [comment]
Just saying that more of these issues should start out as Discussions especially if this much debate will be had. The issue count is ever-growing and issues like this are compounding the problem.

**Comment by DRKSLV** [comment]
@CEbbinghaus yep, totally agree. this issue should be closed imo but yk
can't do that

**Comment by BalooUriza** [comment]
I think keeping the existing functionality but changing what it's called to "content description" is better, given how much it just gets used as a subject line anyway.

**Comment by alper** [comment]
I think one way forward would be to support hashtags in the CW field so that people could standardise on them by convention.

Described here: https://github.com/mastodon/mastodon/issues/20534#issuecomment-1313557524

**Comment by SoniEx2** [comment]
furthermore:

outside of masto, CWs are a prompt for "do you want to leave", not a prompt for "do you want to join"; i.e. they're open by default, they're something you have to make an active decision to not see.

this is, clearly, wrong. we should be following established convention, not making up our own.

**Comment by Nachtpfoetchen** [comment]
uhm actually not. In a lot of places there are Trigger warnings given at the beginning, and it is an active decision to see the content. that is how spoilers in element, discord and telegram behave. you have to click to reveal the content and usually a content warning is given before.

**Comment by SoniEx2** [comment]
yes, those are chat clients.

**Comment by Nachtpfoetchen** [comment]
mastodon is a microblogging site. twitter is one as well. only because mastodon is at places different to twitter, that is not a bad thing. other microblogging sites (like reddit, tumblr, etc.) also have the opportunity to mark sensitive media, subreddits, and posts. One of the reasons i use mastodon over other microblogging sites for years now is because of the good content warning feature and how this feature is used by people.

**Comment by Rexogamer** [comment]
The "established convention" is, frankly, wrong in that case - CWs act (and should continue to act) as a notice *before you open the toot*. It's one thing to say "oh hey, this toot contains content that might upset you or make you feel uncomfortable (e.g. blood), do you want to view it?", and quite another to say "oh right yeah this toot contains blood, sorry for the late notice 🙃".
